packages/Card/Card.jsx

```diff
@@ -71,9 +74,35 @@ const Card = ({ variant, children, fullHeight, spacing, ...rest }) => {
     spacingProps.inset = 4
   }
 
+  const ContainerStyles = imageProps =>
+    handleResponsiveStyles({ position: imageProps?.position }, ({ position }) => {
```

Comment:
We don't support ES2020 yet. Do you mind converting your optional chaining to an explicit check ie.
`imageProps && imageProps.position`

Reply:
Will do!